### PR TITLE
#TRABootCamp-96: [TDD] Implement a method that can validates if the expression containing brackets is valid or not.

### PR DESCRIPTION
**Pull request by iamabdh**
https://trello.com/c/adOtFtcp

### CODE REVIEW

**Review by BaraahAlKalbani — APPROVED** [review]
Great Work by the way 👍🏻

**Review by AnwaarAlbusaidi — APPROVED** [review]
well done, but you miss one test case (Can throw an exception.)

**Comment by iamabdh** [comment]
> well done, but you miss one test case (Can throw an exception.)

updated.

**Review by BaraahAlKalbani — APPROVED** [review]
💯

File: src/main/java/io/code/CodeExample.java [hunk 1]
```diff
@@ -1,12 +1,51 @@
...
+						}
+						else {
+							stack.push(str.charAt(i));
+						}
+					} 
+				i ++;
+			}
+
+		} catch (Exception e) {
+			// TODO: handle exception
```
Comment (CodelineAtyab) [inline src/main/java/io/code/CodeExample.java:47]: We can add some useful message here, so the user knows if something goes wrong.

File: src/test/java/io/code/CodeExampleTest.java [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+		CodeExample cExample = new CodeExample();
+		assertEquals(true, cExample.checkingValidStatementBalance("  "));
+	}
+	
+	@Test
+	@DisplayName("Can throw an exception.")
+	void CheckValidStatementTestCase3() {
+		CodeExample cExample = new CodeExample();
+		assertEquals(Exception.class, cExample.checkingValidStatementBalance(""));
 	}
```
Comment (CodelineAtyab) [inline src/test/java/io/code/CodeExampleTest.java:241]: Great work

**Review by CodelineAtyab — APPROVED** [review]

File: src/main/java/io/code/CodeExample.java (hunk 1 above)
Comment (iamabdh) [reply src/main/java/io/code/CodeExample.java:47]: updated

**Pull request merged**